app/params/params.go

```diff
@@ -1,8 +1,48 @@
 package params
 
+import (
+	"github.com/cosmos/cosmos-sdk/types/address"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/classic-terra/core/v2/types/util"
+)
+
 // Simulation parameter constants
 const (
 	StakePerAccount           = "stake_per_account"
 	InitiallyBondedValidators = "initially_bonded_validators"
 	BondDenom                 = "uluna"
 )
+
+func init() {
```

Comment:
This is dead code - At least I could not find where this function is used. Please, remove

Reply:
Thank you. As discussed with Strath in [this pr](https://github.com/classic-terra/core/pull/414#discussion_r1470745155), this might our mistake for adding it just for testing. We will remove it.